app/dashboard/page.tsx

```diff
@@ -4,12 +4,15 @@ import WeeklyInclusionScore from '@/app/dashboard/_components/cards/WeeklyInclus
 import Trends from '@/app/dashboard/_components/cards/Trends'
 import InclusionScore from '@/app/dashboard/_components/cards/InclusionScore/InclusionScore'
 
+import EntireSection from '@/app/dashboard/_components/RiskAndOpps/EntireSection'
```

Comment:
no empty line above this one and EntireSection isn't a very good name, maybe it should be called RisksAndOpportunitiesSection?